src/main/js/components/SearchList.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable linebreak-style */
```

Comment:
was the linter flagging this file?

Reply:
Accidentally added it while debugging. Removed it though.